velodyne_driver/src/lib/CMakeLists.txt

```diff
@@ -1,12 +1,34 @@
+cmake_minimum_required(VERSION 3.5.0)
+
+# Default to C++14
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
+endif()
+
+find_package(ament_cmake REQUIRED)
+find_package(rmw_implementation REQUIRED)
+find_package(rcl REQUIRED)
+find_package(rclcpp REQUIRED)
+find_package(rcutils REQUIRED)
+find_package(velodyne_msgs REQUIRED)
+find_package(gtest REQUIRED)
+
+# libpcap provides no pkg-config or find_package module:
+set(libpcap_LIBRARIES -lpcap)
+
+include_directories(include
+  ${rclcpp_INCLUDE_DIRS}
+  ${rmw_implementation_INCLUDE_DIRS}
+  ${velodyne_msgs_INCLUDE_DIRS}
+)
+
 add_library(velodyne_input input.cc)
```

Comment:
This should be marked `SHARED` in ROS2 because of how the nodes use the libraries.

Reply:
I got an error when making it `SHARED` but I'll see if I can get it fixed.